File: tasks/prac4_js/task8.js
const prompt = require('prompt-sync')();

console.log("\nЗадача 8: Конвертер температур");
const temp = parseFloat(prompt("Введите температуру: "));
const scale = prompt("Введите шкалу (C/F): ").toUpperCase();

if (scale === "C") {
    const fahrenheit = temp * 9/5 + 32;
    console.log(`${temp}°C = ${fahrenheit.toFixed(1)}°F`);
} else if (scale === "F") {
    const celsius = (temp - 32) * 5/9;
    console.log(`${temp}°F = ${celsius.toFixed(1)}°C`);
} else {
    console.log("Неверная шкала. Используйте C или F");
}

File: tasks/prac4_python/task8.py
temp = float(input("Введите температуру: "))
scale = input("Введите шкалу (C/F): ").upper()

if scale == "C":
    fahrenheit = temp * 9/5 + 32
    print(f"{temp}°C = {fahrenheit:.1f}°F")
elif scale == "F":
    celsius = (temp - 32) * 5/9
    print(f"{temp}°F = {celsius:.1f}°C")
else:
    print("Неверная шкала. Используйте C или F")

# https://pythonworld.ru/osnovy/formatirovanie-strok-operator.html
